zenoh-kotlin/src/commonMain/kotlin/io/zenoh/Zenoh.kt

```diff
@@ -98,15 +99,27 @@ object Zenoh {
     }
 
     /**
-     * Try starting the logs with the level specified under the property 'zenoh.logger'.
+     * Try starting the logs with the level specified under the [LOG_ENV] environment variable.
      *
      * @see Logger
      */
-    fun tryInitLogFromEnv() {
-        val logLevel = System.getProperty("zenoh.logger")
-        if (logLevel != null) {
-            Logger.start(logLevel)
-        }
+    fun tryInitLogFromEnv(): Result<Unit> = runCatching {
+        ZenohLoad
+        val logLevel = System.getenv(LOG_ENV)
+            ?: return Result.failure(Exception("Failure during logs initialization: couldn't find environment variable '$LOG_ENV'."))
+        return Logger.start(logLevel)
+    }
+
+    /**
+     * Try starting the logs with the level specified under the [LOG_ENV] environment variable or by default [defaultLogLevel].
+     *
+     * @param defaultLogLevel A string that must be either "info", "debug", "error", "trace", "warn".
```

Comment:
They are not the only value possible, but I'm not sure we care for now.

Reply:
Somewhere we need to tell the users what are the expected string values for specifying the log level I think.

Reply:
Copy the documentation from here: https://github.com/eclipse-zenoh/zenoh-c/blob/dd5f94aa2a76b9ed3c9c124af03f729e87a52fb8/src/lib.rs#L83

Reply:
I understand now the mechanism of the filters. Added that.